docs/operator_parameters.md

```diff
@@ -0,0 +1,175 @@
+  lacks it using the YAML definition provided by this option. If not defined, a
+  simple definition which contains only the name will be used.
+
+* **watched_namespace**
+  operator watches for Postgres objects in the given namespace. If not
+  specified, the value is taken from the operator namespace. A special '*'
+  value makes it watch all namespaces.
+
+* **pdb_name_format**
+ defines the template for PDB (Pod Disruption Budget) names created by the
+ operator. Default is '*postgres-{cluster}-pdb*', where '*{cluster}*' is the
+ cluster name.
+
+* **secret_name_template**
+  a template for the name of the database user secrets generated by the
+  operator. '{*username*}' is the name of the secret, '{*cluster*'} is the name
+  of the cluster, '{*tprkind*}' is the kind of CRD (formerly known as TPR) and
+  '{*tprgroup*}' is the group of the CRD. Default is
+  '*{username}.{cluster}.credentials.{tprkind}.{tprgroup}*'
+
+* **oauth_token_secret_name**
+  a name of the secret containing the OAuth2 token to pass to the teams API.
+  Default is '*postgresql-operator*'
+
+* **infrastructure_roles_secret_name**
+  name of the secret containing infrastructure roles names and passwords.
+
+### Load balancer related options
+* **db_hosted_zone**
+  DNS zone for the cluster DNS name when the load balancer is configured for
+  the cluster. Default is *db.example.com*. Only used when combined with
+  [external-dns](https://github.com/kubernetes-incubator/external-dns) and the
+  cluster has the load balancer enabled.
+
+* **enable_master_load_balancer**
+  toggles service type load balancer pointing to the master pod of the cluster.
+  Enabled by default and can be overridden by individual cluster settings.
+
+* **enable_replica_load_balancer**
+  toggles service type load balancer pointing to the replica pod of the cluster.
+  Disabled by default and can be overridden by individual cluster settings.
+
+* **master_dns_name_format**
+  defines the DNS name string template for the master load balancer cluster.
+  Default is '*{cluster}.{team}.{hostedzone}*', where '*{cluster}*' is the cluster
+  name, '*{team}*' denotes the team name and '*{hostedzone}*' is the
+  **db_hosted_zone** value.
+
+* **replica_dns_name_format**
+  defines the DNS name string template for the replica load balancer. Default
+  is '*{cluster}-repl.{team}.{hostedzone}*', where '*{cluster}*' is the cluster
+  name, '*{team}*' denotes the team name and '*{hostedzone}*' is the
+  **db_hosted_zone** value.
+ 
+### AWS or GSC interaction
+* **wal_s3_bucket**
+  S3 bucket to use for shipping WAL segments with WAL-E. A bucket has to be
+  present and accessible by Patroni managed pods. At the moment, supported
+  services by Spilo are S3 and GCS.
+
+* **log_s3_bucket**
+  S3 bucket to use for shipping Postgres daily logs. Works only with S3 on AWS.
+  The bucket has to be present and accessible by Patroni managed pods. At the
+  moment Spilo does not yet support this.
```

Comment:
"At the moment" will sound ambiguous in a few weeks. afaik "as of day/month" is better in this regard.